Src/FluentAssertions/Equivalency/INode.cs

```diff
@@ -16,7 +16,10 @@ public interface INode
         /// <summary>
         /// Gets the name of this node.
         /// </summary>
-        string Name { get; }
+        /// <example>
+        /// "Property2"
+        /// </example>
+        string Name { get; set; }
```

Comment:
This is technically a breaking change.

This example will break
```cs
class MyNode : INode
{
    public string Name { get; }
}
```

Reply:
Technically yes, but why would anybody add there own implementation of `INode`. You would have to replace a lot of logic to make that work. Everything is hard-coded to use `Field` or `Property`. And even if somebody would do that, they would override `Node`, which implements this change. So I think it's safe to introduce this change.